mkosi/distributions/serpentos.py

```diff
@@ -0,0 +1,67 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+from collections.abc import Sequence
+
+from mkosi.config import Architecture
+from mkosi.context import Context
+from mkosi.distributions import Distribution, DistributionInstaller, PackageType
+from mkosi.installer.moss import invoke_moss, setup_moss
+from mkosi.log import die
+
+
+class Installer(DistributionInstaller):
+    @classmethod
+    def pretty_name(cls) -> str:
+        return "serpent os"
+
+    @classmethod
+    def filesystem(cls) -> str:
+        return "btrfs"
+
+    @classmethod
+    def package_type(cls) -> PackageType:
+        return PackageType.moss
+
+    @classmethod
+    def default_release(cls) -> str:
+        return "rolling"
+
+    @classmethod
+    def default_tools_tree_distribution(cls) -> Distribution:
+        return Distribution.serpentos
+
+    @classmethod
+    def tools_tree_packages(cls) -> list[str]:
+        return [
+            "systemd",
+            "zstd",
+        ]
+
+    @classmethod
+    def setup(cls, context: Context) -> None:
+        (context.root / "etc").mkdir(exist_ok=True, parents=True)
+
+    @classmethod
+    def install(cls, context: "Context") -> None:
+        setup_moss(context)
+        (context.root / "boot").mkdir(exist_ok=True)
+
+    @classmethod
+    def install_packages(cls, context: Context, packages: Sequence[str], _apivfs: bool = True) -> None:
+        invoke_moss(context, "install", packages)
```

Comment:
`apivfs` is not passed through here. Also please name it `apivfs` like everywhere else.